app/components/case-studies-components/CaseStudyChallengesAndLearnings.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Image from "next/image";
+
+import { CaseStudyHeaderProps } from "@/utils/interfaces";
+import { challengeIcon, greenTick } from "@/public/svg-icons/case-study-icons";
+
+const CaseStudyChallengesAndLearnings: React.FC<CaseStudyHeaderProps> = ({
+  mainProject,
+}) => {
+  return (
+    <section className="flex w-full flex-col items-center bg-white900 px-6 py-9 dark:bg-black200 md:py-[4.5rem]">
```

Comment:
This is to prevent you from getting in the routine of using <section> tags automatically. Keep in mind what your creating and where it will be located inside your HTML. You could consider an `<article> `or `<aside> `tag here.

Reply:
Thanks, I think an article is a really good choice here.